### DISCUSSION TITLE
In-class Thinking Cap

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/21S-ASIAAM-191A#149] [Assignments]
How do you seek to be peers and/or mentors for civil society?

Did the simple act of stating your problem out loud bring any insights?

Did your partner bring anything up that you had not considered?

Was it difficult to avoid giving advice?

Respond to ANY or all of the prompts above in 2-3 sentences.

**Comment by lvalles10** [comment]
I seek to be a peer/mentor for civil society by not only helping empower folks to achieve their goals, but also aspire to hold people I chat about professional development with to truly helping civil society if they've expressed that that's a priority for them in their life. For example, a lot of my friends want to dismantle the nature of oppressive closed source tools, but end up working for big companies in order to make a livable wage. So, I think my role in addition to supporting my friends in the pursuit of their dreams is also holding them accountable to their values.

The act of stating my problem definitely brought out insights about way the I think so frequently in extremes! My partner reminded me that I don't always have to think in a binary form- I can still be involved in communities without taking on an all-consuming leadership role. I hadn't considered the way I was functioning in an all-or-nothing thinking model, which is what my partner helped me be more conscious of.

I like asking questions, so it wasn't difficult to avoid giving advice!

I also just saw the sentence designation; apologies that this went over LOL.

**Comment by ghost** [comment]
Verbalizing my problem allowed me to contextualize the issue I've been running into, the act of which helped me to see my problem more holistically. My partner asked very thought-provoking questions which made me think more extensively about different aspects of my survey I hadn't considered in relation to my problem. It was difficult not to frame my feedback as advice at some points, however, asking questions helped me to understand the problems at hand from a different perspective! I seek to be a peer/mentor for civil society by keeping ethics in mind and remembering to encourage/practice accessibility and accountability.

**Comment by rachan2023** [comment]
I hope to be a peer or a mentor for civil society by talking to my community and listening to their concerns first (instead of providing advice right away) in order to try to empathize with them and let them know that I am hear their voices. The simple act of stating my problem brought a lot of insights! I explained to my partner that I wasn't sure how to implement parts of my project and he asked questions about my current approach and things that I would do differently, so that I can come up with different and new ideas myself. When I was the mentor, it was really difficult preventing myself from giving advice because I want to help them with their problem, but I think listening to their line of thinking is most important.

**Comment by Wny-Duong** [comment]
**How do you seek to be peers and/or mentors for civil society?**
First step's trying to identify a community of civil society that you feel you can have a mutually beneficial relationship with. You only get as much out of a peer/mentorship relationship as you put into it, I think. And even if you don't, it's good to at least have a certain community in mind so if individual relationships break down due to differences of opinions, priorities, etc., you're further introduced to other people in that sector of civil society.

**Did the simple act of stating your problem out loud bring any insights?**
Absolutely. I knew I had an idea of what my issue was (trying to figure out a better way to find a job after graduation), but actually putting it into words for someone else meant that I had to give some extra thought as to what my problem actually was and what kind of things I might've wanted to hear that I haven't heard before that could be useful to me.

**Did your partner bring anything up that you had not considered?**
We started running out of time towards the end but I thought the most of the things my partner brought up were things I already thought of. That's not to say that repeating information isn't helpful, it helps you figure out how others might see their problems and gives you new viewpoints about how you might tackle the problem through someone else's perspectives.

**Was it difficult to avoid giving advice?**
Honestly, not really. I don't really like giving advice unless I'm absolutely confident that what I have to say might be directly related to someone's interests. It's not helpful to give someone advice that's not really applicable to their situation or to suggest something they might've tried because you haven't taken the time to see how their problem-solving process goes. It helps to ask a lot of questions that help nail down what a person's personal goals regarding their issues are before you start considering giving advice. Ideally, the other person might realize a better solution to their problem or have a more specific idea of the kind of advice they'd find useful by that point.

**Comment by epham36** [comment]
I seek to be a peer/mentor for civil society by showing up for my community members! In my conversation, the first problem that came up was one that we both related to! Though no concrete solution or answer was reached, I think the mere act of voicing a challenge one is facing, as well as listening actively and compassionately, validating the other person, helps foster a culture of interdependence between members of civil society in which there is mutual learning, growth, and fulfillment. 

Since I personally try not to give too much advice to folks who come to me with problems, unless they're seeking it out, it wasn't too much of a challenge for me to avoid doing so. I know that sometimes it may be less than helpful, make people feel unheard, it takes away an opportunity for them to find the solution for themselves. However, I could see the challenges with trying to avoid giving advice, especially when it comes to more technical problems that come up when coding or developing software/tech. 

Regardless, having someone talk me through their thought process, goals, and obstacles faced, is usually a helpful way to go to get the ball rolling!

**Comment by vcta99** [comment]
**How do you seek to be peers and/or mentors for civil society?**
A problem I used to have was that I failed to initiate peership/mentorship out of imposter syndrome. Especially at UCLA, I felt that everyone was just so talented and compassionate? To engage with my peers would mean that I believed myself to be on the same playing field, and that was difficult. But I've come to learn that everyone has something new to bring to the table. Like, no one else will ever have your same set of experiences, and that means that no one will ever have the same perspective as you. Tying this back to our breakout room exercise, I really admired Emily prior to ever talking with her one-on-one (Zoom classes be like that lol). I knew she had a knack for thoughtful narrative from a film class we took a class together last Fall, and overall she just had ✨good vibes✨. If I were to fall into bad habits, I would have used my high regard for to diminish my own regard for myself. But instead, I acknowledged that Emily is just really cool, and that her coolness does not take away from my own innate coolness. Ultimately, peership/mentorship is about growing together in coolness.

**Was it difficult to avoid giving advice?**
No, I've always lived by "focus on the person, not the problem". I feel like women are socialized to hold space for others in a way that men aren't. This is kinda unfortunate because I think women are more expected to be an emotional dumping ground for those around them, but I do think it is an important empathetic skill to have. When you focus on the problem and not the person, I find that you risk sounding condescending and might even frustrate the person you are talking to, especially when you haven't taken the time to fully ask about the problem they're facing and are jumping to prescriptive solutions (very mansplaining-esque).

**Comment by brillianz** [comment]
When someone shares one of their issues with me, I always do my best to ask questions to them to make sure that I am understanding correctly. To be an active listener requires that you often times repeat the issue back to them, and making sure that all their concerns are being identified by me. In the arena of civil society, we identify equity and desires of local knowledge systems as foundations for a horizontal dialogue/conversation on how we can plan further. This looks like being facilitators, not owners, of the knowledge that's shared in our position as mentors.

**Comment by rh5140** [comment]
**Was it difficult to avoid giving advice?**

Yes, it was difficult for me to avoid giving advice. I have a bad habit of immediately trying to identify solutions when people share problems/struggles with me without first considering that they may have just wanted to have a space to share their feelings and not to have them get "solved."  "Do you need to vent or do you need advice?" is probably something I should ask more.

During the exercise, I caught myself about to give advice and reframed it as a question. My partner said that questions helped him figure out a solution on his own, and for that I am glad!

**Comment by KenNL42** [comment]
Stating my problem out load can bring some insight for me, as I had to find the right word to convey what my problem is while providing the right amount of background. With this, it helped me think critically and narrow down the problem scope. It benefitted both me and listener to understand what is the core problem that I tried to solve. 
In addition, avoiding giving advice can be difficult because I feel it is easier to tell the answer right away. It also required me to think further beyond the solution, since I also had to come up with a question that leads to solution and provides my perspective. But I feel that this can be more beneficial to listener as I encourage them to find their solution that seem to best fit for their situation, rather than giving my answer that may be biased.

**Comment by annyrviloria** [comment]
**Did your partner bring anything up that you had not considered?**
My conversation with Sam definitely enabled me to think about the practical/logistic particularities of my group project. He offered very helpful questions that helped me to consider the fact that our survey should ideally be easily conducted if using a printed format given that we are allowing legal advocates to complete the survey on behalf of their clients. I also picked up some good tips to facilitate working around people's schedules in the second half of the quarter. Thanks Sam :-)

**Comment by jason-muljadi** [comment]
It was not hard for me to not give advice, and my partner definitely helped me out by sharing new perspectives and ideas. I think the spirit of collaboration that was ingrained in this conversation is necessary and that's how we move towards a civil society. Being ina. civil society means that we consistently challenge each other to be better by bringing in new ideas and actively listening to one another. Without the act of listening, there is so much that we lose out on.

**Comment by avaitt** [comment]
**Did the simple act of stating your problem out loud bring any insights?**
Yes! Putting into words what we want to convey allows us to dwell on the topic and think of a thoughtful answer that addresses a variety of niche aspects of our future. Nothing is for certain, but we certainly have control over our future, and giving thought to the problem helps us better understand what our goals are.  Conveying our deepest goals into words really forces us to dwell on them and truly understand our motives and rationales

**Was it difficult to avoid giving advice?**
Yes! I think sometimes we offer advice without really thinking through someone's situation. The advice can be generic but to really help someone out it takes more than talking - it takes listening conversation and dialogue. By asking questions and understanding another person's point of view, and problems we are forced to think and not offer a half-baked solution that comes from ignorance. I like to think it is better to listen than to talk, and this is often true from what I've observed during conversations.

**Comment by jsxu315** [comment]
Stating my problem didn’t help me realize anything new, but it helped put my problem into perspective so that it became less intimidating. While we didn’t have any “aha!” moments, knowing that someone else had a similar problem did help me feel like I was on the right track. It was a little difficult to avoid giving advice, but this ended up forcing me to consider the problem more deeply and from more angles than if I had tried to immediately jump to a solution. I feel like this type of interaction is helpful when we’re trying to contribute to civil society, because it’s focused on the process of finding something new/helpful rather than the end goal, which can help reduce frustration and help you consider alternate approaches to a project.

**Comment by okcirE** [comment]
Did your partner bring anything up that you had not considered?

Yes. After I shared my problem, my partner would ask a series of questions that would lead and build up to a solution that wasn't simply given away but formed as I answered the questions. I think the questions my partner asked were thought-provoking enough to stir new ideas to solve the problem that I have. Funny thing is that my partner and I have similar problems so we both actually helped ourselves come to a solution of creating daily goals in order to feel satisfied going to bed.

**Comment by ninalong** [comment]
I see being a peer and a mentor as one of the, if not the most impactful thing you can do as a member of civil society. Our interpersonal relationships are one of the areas we can exert the greatest influence/make the greatest change, and I believe that being an active peer and mentor is an absolutely essential part of social justice. Some of my favorite words to live by (that I learned from a mentor!) are that we lift as we climb--as we empower ourselves, we have a duty to empower others.

**Comment by latishasumardy** [comment]
Was it difficult to avoid giving advice?

I felt like it was a bit difficult to avoid giving advice because I'm really used to trying to solve problems right away instead of trying to lead someone to come up with their own answer. I had to really think about the question I wanted to ask, and I had to make sure that my questions would lead them down a useful path of contemplation. This also forced me to really focus on what my partner was thinking, as I needed to fully understand the problem in order to ask questions in regards to it.

**Comment by nnhimichelle** [comment]
I seek to be a peer/mentor for civil society by contributing to the empowerment of people and provide and open and safe space for everyone within the community. From there, I also want to create the assurance that their voices will be heard and listened to is and will always remain a priority within the space. Lastly, I want to emphasize that this space is also overall opportunity for people to grow together where all insights and thoughts are welcome!

Side note: so sorry for late submissions,,,,it's been a time right now LOL
